view/statistics-flow-roles.js

```diff
@@ -141,6 +141,9 @@ exports['statistics-main'] = function () {
 						select(
 							{ id: 'step-status', name: 'status' },
 							list(Object.keys(stepStatuses), function (status) {
+								if (status === 'pending') {
```

Comment:
I'm can't say I'm overjoyed with this solution, as you could simply remove a string from config in line 59.